server/events/templates/multi_project_apply.tmpl

```diff
@@ -1,8 +1,10 @@
 {{ define "multiProjectApply" -}}
 {{ template "multiProjectHeader" . }}
-{{ range $i, $result := .Results }}### {{add $i 1}}. {{ if $result.ProjectName }}project: `{{$result.ProjectName}}` {{ end }}dir: `{{$result.RepoRelDir}}` workspace: `{{$result.Workspace}}`
-{{$result.Rendered}}
+{{ range $i, $result := .Results -}}
+### {{ add $i 1 }}. {{ if $result.ProjectName }}project: `{{ $result.ProjectName }}` {{ end }}dir: `{{ $result.RepoRelDir }}` workspace: `{{ $result.Workspace }}`
+{{ $result.Rendered }}
```

Comment:
change template by following ways

- explicit break line: from `{{ range $i, $result := .Results }}### {{add $i 1}}` to `{{ range $i, $result := .Results -}}\n### {{ add $i 1 }}`
- unified format: from `{{$result.RepoRelDir}}` to `{{ $result.RepoRelDir }}`